pkg/inject/init_proxy.go

```diff
@@ -51,6 +51,7 @@ const proxyInitContainerTemplate = `
 `
 
 type InitContainerTemplateVariables struct {
+	RunAsUser          int64
```

Comment:
add this value in the `securityContext` above.

Reply:
The controller will take the template and convert it to a JSON string, no matter what type declared here will become string when calling json.unmarshal function and in K8s corev1 container runAsUser field does not take String as an input, so I have to add this directly to config instead of using template.